src/app/components/drawer/components/header-action-button.tsx

```diff
@@ -13,22 +14,23 @@ export function HeaderActionButton(props: HeaderActionButtonProps) {
   return (
     <Grid
       _hover={{
-        bg: isWaitingOnPerformedAction ? 'unset' : token('colors.brown.3'),
+        bg: isWaitingOnPerformedAction ? 'unset' : 'accent.component-background-hover',
         cursor: isWaitingOnPerformedAction ? 'unset' : 'pointer',
       }}
       data-testid={HomePageSelectors.DrawerHeaderActionBtn}
       borderRadius="8px"
-      color={token('colors.brown.12')}
+      color="accent.action-primary-default"
+      height="36px"
```

Comment:
We have `36px` in the code a lot, I wonder if we need to add some tokens for size?

Reply:
Yes I think we should, we can make a new task to do it